diffpy/snmf/stretchednmfapp.py

```diff
@@ -18,6 +18,8 @@ def create_parser():
                         help="The directory where the results will be written. Defaults to '<input_directory>/snmf_results'.")
     parser.add_argument('-t', '--data-type', type=str, choices=['powder_diffraction', 'pdf'],
                         help="The type of the experimental data.")
+    parser.add_argument('-l', '--lift', type=float, default=1,
+                        help="The factor that determines how much the data is lifted. By default, the data will be vertically translated to make the minimum value 0.")
```

Comment:
how about "The lifting factor.  Data will be lifted by lifted data = data - min(data)*lift.  Default is 1"

btw, it occurs to me that if the min(data) is positive, the "lift" will "lower" the data to zero if lift = 1 and will lower it to below zero if lift > 1.  This is probably undesirable behavior.  Make sure you have tests for this eventuality  that give the behavior you want.

Reply:
I will make this changes.

The same thought occurred to me. If the min(data) is positive, it's not apparent to me that the data would need to be lifted. My idea is that if the minimum is positive, then do nothing to the data.

Reply:
That is reasonable, though why not allow the user to lift the data if she wants to?

Maybe make sure the lift is a lift,  so it adds the absolute value of min(data)*lift to the data, something like that.

Reply:
Ok.